src/System.Security.Cryptography.X509Certificates/tests/project.json

```diff
@@ -13,6 +13,7 @@
     "System.Security.Cryptography.Algorithms": "4.2.1-beta-24311-02",
     "System.Security.Cryptography.Cng": "4.2.1-beta-24311-02",
     "System.Security.Cryptography.Encoding": "4.0.1-beta-24311-02",
+    "System.Security.Cryptography.OpenSsl": "4.0.1-beta-24311-02",
```

Comment:
Yes not obvious. I needed to add it to get the raw package to be deployed for Linux. Without it, nuget restore was pulling from 1.4 netstandard I believe. FYI this means that the X509 tests for Linux\OsX was running the older OpenSsl package for 1.0 netcoreapp.